src/components/shared/page.tsx

```diff
@@ -189,7 +189,7 @@ function Page({ content, imageSalt, getAssetLocation }: Props): ElementWithResou
                 <title>{content.id}</title>
                 <meta id="twitter-theme" name="twitter:widgets:theme" content="light" />
                 <meta name="viewport" content="initial-scale=1, maximum-scale=5" />
-                <link rel="stylesheet" href="native://fontSize.css" />
+                <link rel="stylesheet" href="https://localhost/fontSize.css" />
```

Comment:
This will (I think) attempt to make a request to `localhost` on port `80` in all circumstances. That won't work locally or when hitting MAPI in a web browser, but we don't expect this request to work anyway in those cases so that's fine.

However, what will this make a request to in the webviews? Is there a server running on `localhost:80` in the apps? If not, is it possible we want `href="/fontSize.css"`, which will make a relative request and pick up the protocol/domain/port from the parent document (in DEV `localhost:8080/fontSize.css`, in PROD `https://mobile.guardianapis.com/fontSize.css`)?

Reply:
On Android there's no server running on `localhost:80` but I think that's OK because we are going to intercept this request in the native layer anyway. That means the webview won't try to make a request to `localhost:80` in the normal way.

We'll need to check what will happen on iOS (@faresite) but my assumption is that there would be no server at `localhost:80` there either and the request would fail (and that won't matter because the base font size is provided differently on iOS).

I thought that for iOS this `localhost` URL might be preferable to using `https://fontSize.css` because, as Fares pointed out, that could trigger a DNS lookup on `fontSize.css` which is unnecessary. AFAIK `.css` is not a real TLD so that DNS lookup will always fail, but they're adding new TLDs left right and centre these days so perhaps another another good reason no to use `https://fontSize.css` is that it might one day actually point to something we don't control!

Reply:
Yeah that's a good point, in URL terms that's TLD `css`, domain `fontSize`.

Is there any reason why we wouldn't want it to be `https://mobile.guardianapis.com/fontSize.css`?

Reply:
`https://mobile.guardianapis.com/fontSize.css` would work fine for Android - we would be able to intercept it just the same way.

On iOS we want this request to fail as quickly/quietly as possible and this is where I think it might not be quite so good? I might be over-thinking it but it seems likely that the `localhost` request will fail pretty quick whereas using `mobile.guardianapis.com` will cause an actual request to be made and to fail with a 404?

Reply:
Yeah, fair point, and stylesheets are render blocking so this isn't ideal.

Ok, I can't see any blockers for doing this. It feels strange to me to request a resource that we _know_ doesn't exist, but I guess we're limited by the platforms here.

Reply:
How do you feel about adding a custom attribute like`data-platform="android"` to tags that are there for one platform only? I could basically remove these tags from the html document native side?

Reply:
That's possible. Only downside I can see is that it's one more piece of HTML manipulation on the iOS side 🤔